docs/RELEASE.md

```diff
@@ -17,21 +17,32 @@ Building involves obtaining a copy of the repository and triggering an automatic
 
 Publishing involves creating a release tag and creating a new Release on GitHub.
 
+NOTE: These steps require your `git remote` to be configured so that that `origin` is your fork and `upstream` is `github.com/kubernetes-sigs/secrets-store-csi-driver`.
```

Comment:
```suggestion
NOTE: These steps require your `git remote` to be configured so that `origin` is your fork and `upstream` is `github.com/kubernetes-sigs/secrets-store-csi-driver`.
```

Reply:
👍